eclair-core/src/test/scala/fr/acinq/eclair/integration/MessageIntegrationSpec.scala

```diff
@@ -81,7 +81,7 @@ class MessageIntegrationSpec extends IntegrationSpec {
     val eventListener = TestProbe()
     nodes("B").system.eventStream.subscribe(eventListener.ref, classOf[OnionMessages.ReceiveMessage])
 
-    val blindedRoute = buildRoute(randomKey(), Seq(IntermediateNode(nodes("A").nodeParams.nodeId), IntermediateNode(nodes("B").nodeParams.nodeId), IntermediateNode(nodes("B").nodeParams.nodeId)), Recipient(nodes("B").nodeParams.nodeId, None)).get
+    val blindedRoute = buildRoute(nodes("A").nodeParams.privateKey, randomKey(), Seq(IntermediateNode(nodes("A").nodeParams.nodeId), IntermediateNode(nodes("B").nodeParams.nodeId), IntermediateNode(nodes("B").nodeParams.nodeId)), Recipient(nodes("B").nodeParams.nodeId, None)).get
```

Comment:
This doesn't test anything interesting since we are no longer the introduction point for this route.

Reply:
Woups, definitely! This shows that something is missing in this test and the other one: we should explicitly assert that the resulting route's introduction node is what we expect. I'll fix that.